src/faketypes.jl

```diff
@@ -39,6 +39,8 @@ function FakeTypeName(@nospecialize(x); justname=false)
         FakeTypeVar(x)
     elseif x isa Core.TypeofBottom
         FakeTypeofBottom()
+    elseif x isa Module && justname
```

Comment:
```suggestion
    elseif x isa Module
```
is more correct, I believe.

Reply:
Sure, thanks!